### ISSUE TITLE
Host a robots.txt file so browsershots.org can be used

### ISSUE THREAD

**Issue by fishouttah20** [lgarron/badssl.com#48]
Wanted to use browsershots.org to hit the dh2048.badssl.com page and see which (if any) browsers had problems with 2048bit parameters. I get an error on their site about not being able to access badssl.com/robots.txt.

Their FAQ (http://browsershots.org/faq#) states: 

Blocked by robots.txt
Browsershots respects the robots.txt standard. If you want, you can explicitly allow Browsershots by adding a section like this to the robots.txt file on your server:
User-agent: Browsershots
Disallow:
Some pages on browsershots.org are also protected, mainly to keep screenshot result pages out of search engines.

**Comment by lgarron** [comment]
https://dh2048.badssl.com/robots.txt now serves the recommended `robots.txt`, but `browsershots.org` still refuses it (as well as other `badssl.com` domains).

Would you mind investigating?

**Comment by fishouttah20** [comment]
This appears to now be working (at least on dh2048.badssl.com) I'm waiting in the queue to confirm there aren't other issues (besides the lack of TLSv1 support requested in another issue).

Thanks!

**Comment by fishouttah20** [comment]
I've confirmed this works on browsershots.org. Thanks for doing this!
